inc/defines.hpp

```diff
@@ -10,6 +10,7 @@
 #include <complex>
 #include <string>
 #include <fftw3.h>
+#include <stdint.h>
```

Comment:
Thanks for the suggestion. I think the following header is more consistent in C++.
```suggestion
#include <cstdint>
```